### ISSUE TITLE
release 1.27.0 Windows zip top-level directory incompatible with the installer

### ISSUE THREAD

**Issue by kolyshkin** [golangci/golangci-lint#1202]
I am trying to upgrade CI for github.com/moby/sys in https://github.com/moby/sys/pull/22 and am getting the following error when trying to install on Windows.

Error log (from e.g. https://github.com/moby/sys/pull/22/checks?check_run_id=823911865, under "Lint")

```console
mkdir -p _build/bin
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b _build/bin v1.27.0
golangci/golangci-lint info checking GitHub for tag 'v1.27.0'
golangci/golangci-lint info found version: 1.27.0 for v1.27.0/windows/amd64
Archive:  golangci-lint-1.27.0-windows-amd64.zip
  inflating: golangci-lint.exe-1.27.0-windows-amd64/LICENSE  
  inflating: golangci-lint.exe-1.27.0-windows-amd64/README.md  
  inflating: golangci-lint.exe-1.27.0-windows-amd64/golangci-lint.exe  
install: cannot stat '/tmp/tmp.13LFEnu37d/golangci-lint-1.27.0-windows-amd64/golangci-lint.exe': No such file or directory
##[error]mingw32-make: *** [Makefile:23: _build/bin/golangci-lint] Error 1
##[error]Process completed with exit code 1.
```

**I believe the cause of the above is the installer expects path `golangci-lint-1.27.0-windows-amd64` while the actual path inside the zip archive is `golangci-lint.exe-1.27.0-windows-amd64` (e.g. there is extra `.exe` in it).**

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

v1.27.0

</details>

<details><summary>Config file</summary>

none
</details>

<details><summary>Go environment</summary>

n/a
</details>

<details><summary>Verbose output of running</summary>

n/a
</details>

**Comment by kolyshkin** [comment]
Indeed, in [golangci-lint-1.27.0-windows-amd64.zip](https://github.com/golangci/golangci-lint/releases/download/v1.27.0/golangci-lint-1.27.0-windows-amd64.zip) the top-level directory is
* `golangci-lint.exe-1.27.0-windows-amd64`
(note the extra `.exe`)

while in the previous 1.26.0 release zip, [golangci-lint-1.26.0-windows-amd64.zip](https://github.com/golangci/golangci-lint/releases/download/v1.26.0/golangci-lint-1.26.0-windows-amd64.zip), it is 
* `golangci-lint-1.26.0-windows-amd64`
(note the absence of `.exe`)

**Comment by kolyshkin** [comment]
@jirfag PTAL

**Comment by iwankgb** [comment]
It seems to be fixed in gorealeser: https://github.com/goreleaser/goreleaser/issues/1500, I'm not sure about the GH Action, though...

**Comment by iwankgb** [comment]
@kolyshkin fixed in [v1.28.0](https://github.com/golangci/golangci-lint/releases/tag/v1.28.0).
